assignments/callbacks.js

```diff
@@ -27,23 +27,39 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return arr.pop();
+  
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x * y);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  let check;
+  if (list.includes(item)) {
+    return check = true;
+  } else {
+    return check = false;
+  }
+}
+
+contains(`Pencil`, items, () => console.log(`test`));
+function bool() {
+  return CB(bool);
```

Comment:
this can be simplified to 

if (list.includes(item)) {
   return cb(true);
  } else {
    return cb(false);
  }

to test it you could simply try `contains('Pencil', items, console.log)`